smrf/framework/CoreConfig.ini

```diff
@@ -318,260 +322,296 @@ coordinates_type:             default = euclidean,
-
-wind_ninja_dxy:               type = int,
-                              description = grid spacing at whcih the WindNinja
-                                            ascii files are output.
-
-wind_ninja_pref:              description = prefix of all outputs from WindNinja
-                                            that matches the topo input to WindNinja.
-
-
-wind_ninja_tz:                default = Europe/London,
-                              description = Time zone that from the WindNinja config.
-
-wind_ninja_roughness:         default = 0.01,
-                              description = the surface roughness used in WindNinja
-                                            generally grass.
-
-wind_ninja_height:            default = 5.0,
-                              description = the output height of wind fileds from
-                                            WindNinja in meters.
+stations:
+type = station list,
+description = Stations to use for distributing wind in M/s
+
+max:
+default = 35.0,
+type = float,
+description = Maximum possible wind in M/s
+
+min:
+default = 0.447,
+type = float,
+description = Minimum possible for wind in M/s
+
+grid_mask:
+default = True,
+type = bool,
+description = Mask the distribution calculations
+
+grid_method:
+default = linear,
+options = [nearest linear cubic],
+description = interpolation method to use for wind
+
+idw_power:
+default = 2.0,
+type = float,
+description = Power for decay of a stations influence in inverse distance
+              weighting
+
+dk_ncores:
+default = 2,
+type = int,
+description = Number of threads to use in the dk calculation
+
+station_peak:
+type = station list,
+description = Name of stations that lie on a peak or a high point
+
+station_default:
+default = 11.4,
+type = float,
+description = Account for sheltered station wind measurements for example 11.4
+              equates to a small forest opening and 0 equates to unsheltered
+              measurements.
+veg_default:
+default = 0.0,
+type = float,
+description = Applies the value to all vegetation not specified
+
+veg_41:
+default = 3.3,
+type = float,
+description = Applies the value where vegetation type equals NLCD class 41
+
+veg_42:
+default = 3.3,
+type = float,
+description = Applies the value where vegetation type equals NLCD class 42
+
+veg_43:
+default = 11.4,
+type = float,
+description = Applies the value where vegetation type equals NLCD class 43
+
+veg_3011:
+default = 3.3,
+type = float,
+description = Applies the value where vegetation equals 3011(Rocky Mountain
+              aspen)
+
+veg_3061:
+default = 3.3,
+type = float,
+description = Applies the value where vegetation equals 3061(mixed aspen)
+
+reduction_factor:
+default = 1.0,
+type = float,
+description = If wind speeds are still off here is a scaling factor
+
+maxus_netcdf:
+type = CriticalFilename,
+description = NetCDF file containing the maxus values for wind
+
+grid_local:
```

Comment:
I am guessing its not. I went in to look it seems like it doesn't get used so I must have just copied and pasted by mistake. Removing grid_local and grid_local_n from wind section

Reply:
I guess a user could use it but I don't know why they would want to.

Reply:
So this is a little more complicated than originally thought. I have added this back in since we use a recipe to manage this when any distribution == grid